packages/expo-dev-launcher/bundle/screens/SettingsScreen.tsx

```diff
@@ -149,6 +152,26 @@ export function SettingsScreen() {
                 {touchGestureEnabled && <CheckIcon />}
               </Row>
             </Button.FadeOnPressContainer>
+
+            <Divider />
+
+            <Button.FadeOnPressContainer
+              bg="default"
+              roundedBottom="large"
+              roundedTop="none"
+              onPress={() => setButtonGestureEnabled(!buttonGestureEnabled)}
+              accessibilityState={{ checked: buttonGestureEnabled }}>
+              <Row px="medium" py="small" bg="default">
+                {/* TODO: Replace with a floating button icon, this is kind of close enough for now */}
+                <ToolbarOverlayIcon />
```

Comment:
honestly this icon looks pretty close to what we'd want even... but maybe we should make a new one

Reply:
todo: hide this on ios